### DISCUSSION TITLE
Welcome to indian-banks Discussions!

### DISCUSSION THREAD

**Discussion by praveenpuglia** [praveenpuglia/indian-banks#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akash3gtm** [comment]
Hi Praveen,

Akash Gautam here, almost 3 years working with open source regarding react, expo, next, react native. Mostly playing with universal UI libraries, and now working with an edtech company and still contributing on the side.

First of all.. thank you for this man. 😄 

I started using this recently, and just want to ask. How do you get the images?.. do you just inspect their websites and copy from there or do you manually create them.. or something else entirely? I would love to help with this.

**Comment by praveenpuglia** [comment]
Hey!
Thanks for reaching out and I am glad this repo is useful to someone. 

The process is a mix of bunch of things. 
- I first go to the official website to see if they have something. These days, most websites use an SVG logo so it's easy to copy paste from there. But these are mostly full logos. 
- For Symbols( small logos ) I try and see if their favicon setup contains that as SVG or not. If not, most logos have the symbol embedded in their main logo itself. I then use Inkscape to cut it out and save it. 
- A bunch of times banks have Press Kits available on their website. 
- An underrated source for vector logos is WikiMedia
- If I don't find anything then I just google it. I do end up finding some Figma that has the latest logo / symbol and then use it from there.

**Reply by akash3gtm** [reply]
Cool man, will see If I can contribute somehow for the remaining banks.

**Reply by praveenpuglia** [reply]
Looking forward to it! Thanks!
